feg/gateway/Makefile

```diff
@@ -15,6 +15,9 @@ download:
 install:
 	go install ./... magma/gateway/services/magmad
 
+install_envoy_controller:
```

Comment:
Why is this in feg?

Reply:
lte/ didn't have a go/ setup so as a quick solution put it into feg(same as aaa)

Reply:
As we are getting more and more go services on `lte/` should we invest time in creating a `go/` folder? Maybe ask more ppl here

Reply:
Actually this should be the last go service for a while(we have a decision to only use c++/python from now on) the envoy service library was only available in go or java so this the decision.

Maybe we can get by with it there for now, as its lower pri